File: Week1/MathProblem/MultiplicationTable.java
import java.util.ArrayList;

public class MultiplicationTable {

/*
 * Create the multiplication table of a given number N and return the table as an array.
 * https://www.geeksforgeeks.org/problems/print-table0303/1
 * T(c) -> O(1), S(c) -> O(1)
 */

    static ArrayList<Integer> getTable(int N){
        // code here
        ArrayList<Integer> numList = new ArrayList<>();
        for(int i=1;i<=10;i++)
        {
            numList.add(N*i);
        }
        return numList;
    }

    public static void main(String[] args) {
        int N = 5;
        ArrayList<Integer> table = getTable(N);
        System.out.println(table); 
    }
    
}

File: Week1/MathProblem/Python/MultiplicationTable.py
"""
    Create the multiplication table of a given number N and return the table as an array.
    https://www.geeksforgeeks.org/problems/print-table0303/1
"""

def getTable(N):
    """
        T(c) -> O(1), S(c) -> O(1)
    """
    res = []
    i = 1
    while i<=10:
        res.append(N*i)
        i += 1
    
    return res

print(getTable(9))